acceptancetests/assess_cross_model_relations.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+"""Functional tests for Cross Model Relation (CMR) functionality.
+
+This test exercises the CMR Juju functionality which allows applications to
+communicate between different models (including across controllers/clouds).
+
+The outline of this feature can be found here[1].
+
+This test will exercise the following aspects:
+  - Ensure a user is able to create an offer of a relation of an application
+    including:
+      - The offer appears in the list-offers output
+      - The user is able to name the offer
+      - The user is able to remove the offer
+
+The above feature tests will be run on:
```

Comment:
As a note - the test will ultimately need to cover the situation where both a requires and provides endpoint is offered.
Also, it's WIP for sure and a good start, but the test as of now is not really a *complete* test any real cmr scenario - it doesn't verify that the workloads can talk to each other. It verifies that info can be swapped over relation data but it needs to check that the correct firewall ports are opened etc to allow workload-workload traffic. Ultimately this will need to cover various ingress/egress situations etc.